snips-nlu-lib/src/intent_classifier/log_reg_intent_classifier.rs

```diff
@@ -6,14 +6,16 @@ use std::sync::Arc;
 use itertools::Itertools;
 use ndarray::prelude::*;
 use serde_json;
+use snips_nlu_ontology::IntentClassifierResult;
 
 use models::IntentClassifierModel;
 use errors::*;
+use nlu_engine::load_engine_shared_resources;
```

Comment:
This import is only used in test, so we should move it there.

Reply:
it's used in the `inject` method